### PR TITLE
Batch automation: Formatting fixes

### PR DESCRIPTION
**Pull request by bobbytreed**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.1 document
- [ ] Impacts 6.0 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[developer/module/how-to-write-a-powershell-binary-module.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/how-to-write-a-powershell-binary-module.md) | :white_check_mark:Succeeded | |
[developer/module/how-to-write-a-powershell-module-manifest.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/how-to-write-a-powershell-module-manifest.md) | :white_check_mark:Succeeded | |
[developer/module/how-to-write-a-powershell-script-module.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/how-to-write-a-powershell-script-module.md) | :white_check_mark:Succeeded | |
[developer/module/how-updatable-help-works.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/how-updatable-help-works.md) | :white_check_mark:Succeeded | |
[developer/module/importing-a-powershell-module.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/importing-a-powershell-module.md) | :white_check_mark:Succeeded | |
[developer/module/installing-a-powershell-module.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/installing-a-powershell-module.md) | :white_check_mark:Succeeded | |
[developer/module/modifying-the-psmodulepath-installation-path.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/modifying-the-psmodulepath-installation-path.md) | :white_check_mark:Succeeded | |
[developer/module/naming-help-files.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/naming-help-files.md) | :white_check_mark:Succeeded | |
[developer/module/supporting-online-help.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/supporting-online-help.md) | :white_check_mark:Succeeded | |
[developer/module/supporting-updatable-help.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/supporting-updatable-help.md) | :white_check_mark:Succeeded | |
[developer/module/understanding-a-windows-powershell-module.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/understanding-a-windows-powershell-module.md) | :white_check_mark:Succeeded | |
[developer/module/updatable-help-authoring-step-by-step.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/updatable-help-authoring-step-by-step.md) | :white_check_mark:Succeeded | |
[developer/module/updatable-help-overview.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/updatable-help-overview.md) | :white_check_mark:Succeeded | |
[developer/module/writing-a-windows-powershell-module.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/writing-a-windows-powershell-module.md) | :white_check_mark:Succeeded | |
[developer/module/writing-help-for-windows-powershell-modules.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/writing-help-for-windows-powershell-modules.md) | :white_check_mark:Succeeded | |
[developer/provider/windows-powershell-provider-quickstart.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/provider/windows-powershell-provider-quickstart.md) | :white_check_mark:Succeeded | |
[developer/provider/writing-a-container-provider.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/provider/writing-a-container-provider.md) | :white_check_mark:Succeeded | |
[developer/provider/writing-a-navigation-provider.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/provider/writing-a-navigation-provider.md) | :white_check_mark:Succeeded | |
[developer/provider/writing-a-windows-powershell-provider.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/provider/writing-a-windows-powershell-provider.md) | :white_check_mark:Succeeded | |
[developer/provider/writing-an-item-provider.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/provider/writing-an-item-provider.md) | :white_check_mark:Succeeded | |
[developer/webservices/adding-resources-to-a-management-odata-web-service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/webservices/adding-resources-to-a-management-odata-web-service.md) | :white_check_mark:Succeeded | |
[developer/webservices/associating-management-odata-entities.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/webservices/associating-management-odata-entities.md) | :white_check_mark:Succeeded | |
[developer/webservices/authoring-the-mof-schema-file-for-a-management-odata-web-service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/webservices/authoring-the-mof-schema-file-for-a-management-odata-web-service.md) | :white_check_mark:Succeeded | |
[developer/webservices/authoring-the-web-config-file-for-a-management-odata-web-service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/webservices/authoring-the-web-config-file-for-a-management-odata-web-service.md) | :white_check_mark:Succeeded | |
[developer/webservices/authoring-the-xml-schema-file-for-a-management-odata-web-service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/webservices/authoring-the-xml-schema-file-for-a-management-odata-web-service.md) | :white_check_mark:Succeeded | |
[developer/webservices/configuring-role-based-authorization.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/webservices/configuring-role-based-authorization.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C8%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806081555113446-2539%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=SlUdJPGZe3r431D9tNU0SkzkMnJDqX03uhAY0fLShdo%3D&st=2018-06-08T16%3A00%3A38Z&se=2018-07-09T16%3A05%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: developer/module/supporting-online-help.md [hunk 1]
```diff
@@ -1,101 +1,111 @@
...
+For workflows that are written in the Windows PowerShell language, add an **.ExternalHelp** comment directive to the workflow code. The value of the directive must be a URI that begins with "http" or "https".
+
+> [!NOTE]
+> The HelpUri property is not supported for XAML-based workflows in Windows PowerShell.
+
+The following code shows the .ExternalHelp directive in a workflow file.
+
+```powershell
+# .ExternalHelp "http://go.microsoft.com/fwlink/?LinkID=138338"
+```
```
Comment (sdwheeler) [inline developer/module/supporting-online-help.md]: Control character

File: developer/webservices/configuring-role-based-authorization.md [hunk 2]
```diff
@@ -1,104 +1,107 @@
...
+
+   ```xml
+   <User Name="UserName" GroupName="AdminGroup" AuthenticationType="Basic" DomainName="DomainName"/>
+   ```
+
+3. Directly after the XML added in the previous step, add the following XML:
+
+   ```xml
+   <User Name="UserName" GroupName="NonAdminGroup" AuthenticationType="Basic" DomainName="DomainName"/>
+   ```
```
Comment (sdwheeler) [inline developer/webservices/configuring-role-based-authorization.md]: Control character

File: developer/provider/windows-powershell-provider-quickstart.md [hunk 3]
```diff
@@ -1,167 +1,174 @@
...
+      if (accessDBPSDriveInfo == null)
+      {
+         return null;
+      }
+
+      accessDBPSDriveInfo.Connection.Close();
+
+      return accessDBPSDriveInfo;
+    }
+```
```
Comment (sdwheeler) [inline developer/provider/windows-powershell-provider-quickstart.md]: Control character

File: developer/module/how-to-write-a-powershell-script-module.md [hunk 4]
```diff
@@ -1,227 +1,231 @@
...
+    $displayCalendar = " 'n" + $padding + $header + "'n " + $calendar
+    $displayCalendar.TrimEnd()
+
+    ## Move to the next month.
+    $start = $start.AddMonths(1)
+
+}
+}
+Export-ModuleMember -Function Show-Calendar
+```
```
Comment (sdwheeler) [inline developer/module/how-to-write-a-powershell-script-module.md]: Control character

File: developer/module/supporting-online-help.md (hunk 1 above)
Comment (bobbytreed) [reply developer/module/supporting-online-help.md]: Fixed this

File: developer/webservices/configuring-role-based-authorization.md (hunk 2 above)
Comment (bobbytreed) [reply developer/webservices/configuring-role-based-authorization.md]: Fixed this

File: developer/provider/windows-powershell-provider-quickstart.md (hunk 3 above)
Comment (bobbytreed) [reply developer/provider/windows-powershell-provider-quickstart.md]: Fixed this

File: developer/module/how-to-write-a-powershell-script-module.md (hunk 4 above)
Comment (bobbytreed) [reply developer/module/how-to-write-a-powershell-script-module.md]: Fixed this

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[developer/module/how-to-write-a-powershell-binary-module.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/how-to-write-a-powershell-binary-module.md) | :white_check_mark:Succeeded | |
[developer/module/how-to-write-a-powershell-module-manifest.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/how-to-write-a-powershell-module-manifest.md) | :white_check_mark:Succeeded | |
[developer/module/how-to-write-a-powershell-script-module.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/how-to-write-a-powershell-script-module.md) | :white_check_mark:Succeeded | |
[developer/module/how-updatable-help-works.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/how-updatable-help-works.md) | :white_check_mark:Succeeded | |
[developer/module/importing-a-powershell-module.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/importing-a-powershell-module.md) | :white_check_mark:Succeeded | |
[developer/module/installing-a-powershell-module.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/installing-a-powershell-module.md) | :white_check_mark:Succeeded | |
[developer/module/modifying-the-psmodulepath-installation-path.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/modifying-the-psmodulepath-installation-path.md) | :white_check_mark:Succeeded | |
[developer/module/naming-help-files.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/naming-help-files.md) | :white_check_mark:Succeeded | |
[developer/module/supporting-online-help.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/supporting-online-help.md) | :white_check_mark:Succeeded | |
[developer/module/supporting-updatable-help.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/supporting-updatable-help.md) | :white_check_mark:Succeeded | |
[developer/module/understanding-a-windows-powershell-module.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/understanding-a-windows-powershell-module.md) | :white_check_mark:Succeeded | |
[developer/module/updatable-help-authoring-step-by-step.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/updatable-help-authoring-step-by-step.md) | :white_check_mark:Succeeded | |
[developer/module/updatable-help-overview.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/updatable-help-overview.md) | :white_check_mark:Succeeded | |
[developer/module/writing-a-windows-powershell-module.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/writing-a-windows-powershell-module.md) | :white_check_mark:Succeeded | |
[developer/module/writing-help-for-windows-powershell-modules.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/module/writing-help-for-windows-powershell-modules.md) | :white_check_mark:Succeeded | |
[developer/provider/windows-powershell-provider-quickstart.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/provider/windows-powershell-provider-quickstart.md) | :white_check_mark:Succeeded | |
[developer/provider/writing-a-container-provider.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/provider/writing-a-container-provider.md) | :white_check_mark:Succeeded | |
[developer/provider/writing-a-navigation-provider.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/provider/writing-a-navigation-provider.md) | :white_check_mark:Succeeded | |
[developer/provider/writing-a-windows-powershell-provider.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/provider/writing-a-windows-powershell-provider.md) | :white_check_mark:Succeeded | |
[developer/provider/writing-an-item-provider.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/provider/writing-an-item-provider.md) | :white_check_mark:Succeeded | |
[developer/webservices/adding-resources-to-a-management-odata-web-service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/webservices/adding-resources-to-a-management-odata-web-service.md) | :white_check_mark:Succeeded | |
[developer/webservices/associating-management-odata-entities.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/webservices/associating-management-odata-entities.md) | :white_check_mark:Succeeded | |
[developer/webservices/authoring-the-mof-schema-file-for-a-management-odata-web-service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/webservices/authoring-the-mof-schema-file-for-a-management-odata-web-service.md) | :white_check_mark:Succeeded | |
[developer/webservices/authoring-the-web-config-file-for-a-management-odata-web-service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/webservices/authoring-the-web-config-file-for-a-management-odata-web-service.md) | :white_check_mark:Succeeded | |
[developer/webservices/authoring-the-xml-schema-file-for-a-management-odata-web-service.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/webservices/authoring-the-xml-schema-file-for-a-management-odata-web-service.md) | :white_check_mark:Succeeded | |
[developer/webservices/configuring-role-based-authorization.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b021/developer/webservices/configuring-role-based-authorization.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C8%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806081738402832-2539%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=dWFhxiniaY70LDhMmzcVgnSb4FXurLSyDnTSVCsoUJM%3D&st=2018-06-08T17%3A43%3A53Z&se=2018-07-09T17%3A48%3A53Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**